### PR TITLE
Con 899 azure viz

### PR DESCRIPTION
**Pull request by ajayshekar**
Same viz query for all azure rules for now.

### CODE REVIEW

File: services/config.rb [hunk 1]
```diff
@@ -31,6 +31,17 @@
...
+  meta_rule_visualize <<~QUERY
+  visualize(func: uid(<%= violation_uid %>)) {
+  	<%= default_predicates %>
+  	relates_to {
+  		<%= default_predicates %>
+  		relates_to @filter(NOT uid(<%= violation_uid %>)){
+  			<%= default_predicates %>
+  		}
+  	}
+  }
```
Comment (armaneous) [inline services/config.rb]: Needs to be wrapped in `{}`, it's a graphql thing.

File: services/config.rb [hunk 2]
```diff
@@ -31,6 +31,17 @@
     }
   }
   QUERY
+  meta_rule_visualize <<~QUERY
+  visualize(func: uid(<%= violation_uid %>)) {
+  	<%= default_predicates %>
```
Comment (armaneous) [inline services/config.rb]: Mixed-case indents, strings and tabs. Stick to strings (we're doing 2-space indents here).

**Comment by armaneous** [comment]
My comments are applicable to all viz queries.

File: services/config.rb (hunk 1 above)
Comment (ajayshekar) [reply services/config.rb]: Great catch Andrew! Thanks!

Comment (ajayshekar) [reply services/config.rb]: @armaneous let me know if the indentation looks okay now :)

**Pull request merged**